documentation/collectionrules/customshortcuts.md

```diff
@@ -0,0 +1,294 @@
+# Custom Shortcuts
+
+Custom Shortcuts allow users to design reusable collection rule components to decrease configuration verbosity, reduce duplication between rules, and speed up the process of writing complex scenarios.
+
+Custom Shortcuts associate a name with a single Filter, Trigger, Action, or Limit; this name can then be used throughout configuration to represent the use of that Filter/Trigger/Action/Limit. This is ideal for scenarios where multiple collection rules re-use the same functionality, allowing the author to write/edit the configuration in a single place.
+
+You can easily translate existing configuration to Custom Shortcuts using the format in the following sample; once defined, the Custom Shortcut is referenced by its name in the collection rule.
+
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "CustomShortcuts": {
```

Comment:
Templates

Reply:
There really isn't any template-y about the "built-in shortcuts". Those are just other triggers that happen to be event counter based. A template is typically something where you take the content as-is and do some token replacement, if any.